contracts/sol6/SanityRatesMaxGasPrice.sol

```diff
@@ -0,0 +1,61 @@
+pragma solidity 0.6.6;
+
+import "./utils/Utils5.sol";
+import "./utils/WithdrawableNoModifiers.sol";
+
+
+contract SanityRatesMaxGasPrice is WithdrawableNoModifiers, Utils5 {
+    mapping(address => uint256) public tokenRate;
+    mapping(address => uint256) public reasonableDiffInBps;
+    uint256 internal maxGasPriceValue = 50 wei * 1000000000; // 50 gwei
+
+    event SanityMaxGasPriceSet(uint256 maxGasPrice);
+
+    constructor(address _admin) public WithdrawableNoModifiers(_admin) {
+        require(_admin != address(0));
+        admin = _admin;
+    }
+
+    function setReasonableDiff(IERC20[] memory srcs, uint256[] memory diff) public {
+        onlyAdmin();
+        require(srcs.length == diff.length);
+        for (uint256 i = 0; i < srcs.length; i++) {
+            require(diff[i] <= 100 * 100);
+            reasonableDiffInBps[address(srcs[i])] = diff[i];
+        }
+    }
+
+    function setMaxGasPrice(uint256 _maxGasPrice) public {
```

Comment:
please add units.
setMaxGasPriceGwei(

I think gwei is easier to use.
but wei also OK
important to note which units are used

Reply:
Named it `setMaxGasPriceWei` since the input must be in wei, similar to how we set maxGasPrice in network.